build.sbt

```diff
@@ -51,30 +46,6 @@ Global / concurrentRestrictions += Tags.limit(ScalaJSTags.Link, 2)
 // Uncomment for local gmp testing
 // ThisBuild / resolvers += "Local Maven Repository" at "file://"+Path.userHome.absolutePath+"/.m2/repository"
 
-// Settings to use git to define the version of the project
-def versionFmt(out: sbtdynver.GitDescribeOutput): String = {
-  val dirtySuffix = if (out.dirtySuffix.mkString("", "").nonEmpty) {
-    "-UNCOMMITED"
-  } else {
-    ""
-  }
-  s"-${out.commitSuffix.sha}$dirtySuffix"
-}
-
-def fallbackVersion(d: java.util.Date): String = s"HEAD-${sbtdynver.DynVer.timestamp(d)}"
-
-val dateFormatter = java.time.format.DateTimeFormatter.BASIC_ISO_DATE
-
-inThisBuild(
```

Comment:
In applications which we won't release as libraries we have been using this version format as it is displayed to the end users

I'm not sure if we want to change it, or maybe it should be push down to the part where the actual releases are made

Reply:
Huh. So the version of your published artifacts doesn't match your tags?

Reply:
Observe and explore are end user applications, we don't publish them as artifacts but as full apps

In the case of explore it is just web assets, js, css, etc

Observe is released as a tarball